src/ensembl/src/content/app/browser/services/browser-messaging-service/browser-messaging-service.test.ts

```diff
@@ -15,16 +15,20 @@
  */
 
 import { BrowserMessagingService } from './browser-messaging-service';
+import { zmenuEnterMessage as dummyMessageCreator } from './browser-message-creator';
```

Comment:
Aren't we abusing the `zmenuEnterMessage` here? Let's write the dummyMessageCreator in the test file itself.